pkgs/development/node-packages/main-programs.nix

```
@@ -6,6 +6,7 @@
   "@microsoft/rush" = "rush";
   "@webassemblyjs/cli-1.11.1" = "wasm2wast";
   "@withgraphite/graphite-cli" = "gt";
+  "@protobuf-ts/plugin" = "protoc-gen-ts";
```

I believe this list should be in alphabetical order
```suggestion
  "@microsoft/rush" = "rush";
  "@protobuf-ts/plugin" = "protoc-gen-ts";
  "@webassemblyjs/cli-1.11.1" = "wasm2wast";
  "@withgraphite/graphite-cli" = "gt";
```